src/cdk/schematics/ng-update/devkit-migration-rule.ts

```diff
@@ -91,7 +91,9 @@ export function createMigrationSchematicRule(
       const testTsconfigPath = getTargetTsconfigPath(project, 'test');
 
       if (!buildTsconfigPath && !testTsconfigPath) {
-        logger.warn(`Could not find TypeScript project for project: ${projectName}`);
+        logger.warn(
+          `Skipping migration for project ${projectName} due to missing TypeScript project configuration in angular.json.`,
```

Comment:
this is super nit, but is it really good to say "missing"? seems like it would be more accurate to say that we could not find it. e.g. 

```suggestion
          `Skipping migration for project ${projectName}. Unable to determine `tsconfig.json` file in workspace config.`,
```

Reply:
My whole commit is a little nit ;-). But yeah, I absolutely agree with your suggestion. Updated the commit.

Reply:
haha, no it's great! definitely was ambiguous before. Thank you!!